nginx/nginx.conf

```diff
@@ -29,7 +29,48 @@ http {
 
     keepalive_timeout  65;
 
-    #gzip  on;
+    gzip              on;
+    gzip_disable      "msie6";
+    # gzip requests proxied from the load balancer
+    gzip_proxied      any;
```

Comment:
It's more standard to *not* gzip proxied requests as you generally don't want to gzip things that will be cached upstream. In our case we don't have any caching set up upstream yet so I'm just gzip-ing everything.